test/groovy/com/sap/piper/PiperGoUtilsTest.groovy

```diff
@@ -17,153 +17,153 @@ import static org.junit.Assert.assertThat
 
 class PiperGoUtilsTest extends BasePiperTest {
 
-    public ExpectedException exception = ExpectedException.none()
```

Comment:
The format should have been fine. What is the change?

Reply:
fixed